src/components/posts/PostCard.jsx

```diff
@@ -0,0 +1,188 @@
+import { useState, useEffect } from 'react'
+import styled from 'styled-components'
+import { useUser } from 'libs/recoil/hooks'
+import { Neumorphism } from 'components/commons/Neumorphism'
+import { Glassmorphism } from 'components/commons/Glassmorphism'
+import { getPostData, changeFavorite } from 'apis/firebase/hooks'
+import { PostUserCard } from 'components/posts/PostUserCard'
+import ImageGallery from 'react-image-gallery'
+import 'react-image-gallery/styles/css/image-gallery.css'
+import { Slider } from 'components/commons/Slider'
+import dayjs from 'dayjs'
+import FavoriteIcon from '@mui/icons-material/Favorite'
+import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder'
```

Comment:
```suggestion
import { Favorite, FavoriteBorder } from '@mui/icons-material'
```
実はMUIのiconsもこう書ける気がします